packages/search/addon/components/navi-search-bar.js

```diff
@@ -0,0 +1,3 @@
+import Component from '@glimmer/component';
+
+export default class NaviSearchBarComponent extends Component {}
```

Comment:
Make sure we can still override these in the app, you may or may not have to use [`setComponentTemplate`](https://github.com/emberjs/rfcs/blob/master/text/0481-component-templates-co-location.md#low-level-primitives) on the component in order to be able to override it

Reply:
I think we won't need to worry about this, the docs say 

> The low-level setComponentTemplate and getComponentTemplate APIs are not intended to be called by end-users directly, even though they will be public and part of our semver stability guarantee. Instead, they are meant to be used primarily in the output emitted by build tools. At build time, any template files found in {app,addon}/components will be inlined into the component's JavaScript file and removed from the build output.

If the subclassed component has a template with the same name it should pick this one up instead of the inherited template.

Reply:
I ended up having issues with template co-location/glimmer components when using it in an addon since glimmer components don't have a `layout` field like classic ember components. Although since these have no implementation I guess they have to be extended anyways so maybe not an issue